orderlyweb_api/orderly_web_api.py

```diff
@@ -9,21 +11,40 @@ def __init__(self, base_url, token):
         response = requests.post(auth_url, headers=headers)
         if response.status_code != 200:
             msg = 'Unexpected status code: {}. Unable to authenticate.'
-            raise Exception(msg.format(response.status_code))
+            raise OrderlyWebResponseError(msg.format(response.status_code),
```

Comment:
Might it be more useful to just pass through the error message from orderly web? These should be readable, informative things like "Invalid token" or "Missing required permissions: ..."